OracleUnifiedDirectorySM/README.md

```diff
@@ -0,0 +1,198 @@
+Oracle Unified Directory Service Manager (OUDSM) on Docker
+==========================================================
+
+## Contents
+
+1. [Introduction](#1-introduction-1)
+2. [Hardware and Software Requirements](#2-hardware-and-software-requirements)
+3. [Prerequisites](#3-prerequisites)
+4. [Building OUDSM Docker Image](#4-loading-or-building-oudsm-docker-image)
+5. [Preparing to Run OUDSM Docker Image](#5-preparing-to-run-oudsm-docker-image)
+6. [Running OUDSM Docker Container](#6-running-oudsm-docker-container)
+
+# 1. Introduction
+This project offers Dockerfile and scripts to build an Oracle Unified Directory Service Manager image based on 12cPS4 (12.2.1.4.0) release. Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+This Image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create FMW Infrastructure domain with OUDSM application deployed in container with OUDSM deployed which can be targeted for development and testing.
+
+***Image***: oracle/oudsm:12.2.1.4.0
+
+# 2. Hardware and Software Requirements
+Oracle Unified Directory Service Managedr Docker Image has been tested and is known to run on following hardware and software:
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 18.03 or higher | docker version            |
+
+# 3. Prerequisites
+
+## 3.1 Pulling the Oracle JDK (Server JRE) base image
+You can pull the Oracle Server JRE 8 image from the [Oracle Container Registry](https://container-registry.oracle.com). When pulling the Server JRE 8 image, re-tag the image so that it works with the dependent Dockerfiles which refer to the JRE 8 image through oracle/serverjre:8.
+
+**IMPORTANT**: Before you pull the image from the registry, please make sure to log-in through your browser with your SSO credentials and ACCEPT "Terms and Restrictions".
+
+1. Sign in to [Oracle Container Registry](https://container-registry.oracle.com). Click the **Sign in** link which is on the top-right of the Web page.
+2. Click **Java** and then click on **serverjre**.
+3. Click **Accept** to accept the license agreement.
+4. Use following commands to pull Oracle Fusion Middleware infrastructure base image from repository :
+
+        
+        $ docker login container-registry.oracle.com
+        $ docker pull container-registry.oracle.com/java/serverjre:8
+        $ docker tag container-registry.oracle.com/java/serverjre:8 oracle/serverjre:8
+
+## 3.2 Pulling Oracle FMW Infrastructure 12.2.1.4.x image
```

Comment:
Likewise you should link to the `fmw-infrastructure` image build documentation.

Reply:
With usage of imagetool, fmw-infrastructure image would not be required to be built separately.

Reply:
But what about customers who want to build it themselves?

Reply:
Added link to Java and FMW Infra Build documentation